### DISCUSSION TITLE
I made a new K8s Container

### DISCUSSION THREAD

**Discussion by RamboRogers** [shadowsocks/shadowsocks-rust#977] [Show and tell]
Hello Team,

I built this for my K8s cluster.  Seems to work pretty well so I figured I would share here.  I'd like to copy into the examples folder to help others.

[shadowsocks-server-k8s](https://github.com/RamboRogers/shadowsocks-server-k8s)

**Comment by zonyitoo** [comment]
Feel free to open a PR if you will. Actually there already a compose configuration file `k8s` folder.
